src/hooks/navigation/useScrollHandler.tsx

```diff
@@ -11,16 +11,35 @@ import { useWindowContext } from 'hooks';
 import { useAudioPlayer, useEffectOnce, useEventListener } from '../';
 import { IpodEvent } from 'utils/events';
 
+const getInitIndex = (
```

Comment:
This is a great idea! I hadn't thought to default to the currently selected item in the list. Perhaps we could add a comment to this just to describe what it's accomplishing?